assets/src/edit-story/components/helpCenter/index.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { useState } from 'react';
+import { useFeatures } from 'flagged';
+import styled, { ThemeProvider } from 'styled-components';
+/**
+ * Internal dependencies
+ */
+import { theme as dsTheme, ThemeGlobals } from '../../../design-system';
+import { Toggle } from './toggle';
+
+const Wrapper = styled.div`
+  position: absolute;
+  bottom: 16px;
+  left: 12px;
+  z-index: 10;
+
+  @media ${({ theme }) => theme.breakpoint.tablet} {
+    bottom: 24px;
+    left: 24px;
+  }
+
+  @media ${({ theme }) => theme.breakpoint.destop} {
+    bottom: 28px;
+    left: 34px;
```

Comment:
This looks kinda weird in the editor, maybe because of the misalignment with the "version" line
![Screen Shot 2021-01-26 at 10 51 23 AM](https://user-images.githubusercontent.com/41136059/105883777-7967b300-5fc4-11eb-9dc9-bdf9bdc6732b.png)

Reply:
yea I don't think the new designs have that version label. 🤷 

I'll ping the design channel and can update it in a later task so it's not blocking progress here